main.go

```diff
@@ -94,14 +96,18 @@ func (t *tfnotify) Run() error {
 			Owner:   t.config.Notifier.Github.Repository.Owner,
 			Repo:    t.config.Notifier.Github.Repository.Name,
 			PR: github.PullRequest{
-				Revision: ci.PR.Revision,
-				Number:   ci.PR.Number,
-				Title:    t.context.String("title"),
-				Message:  t.context.String("message"),
+				Revision:              ci.PR.Revision,
+				Number:                ci.PR.Number,
+				Title:                 t.context.String("title"),
+				Message:               t.context.String("message"),
+				DestroyWarningTitle:   t.context.String("destroy-warning-title"),
```

Comment:
Of course, good. But I think it's enough that this can be set by tfnotify config. The merits of this case is less compared with "Title" case. What do you think?


I know "Template" type is a basically interface. So to satisfy that interface, we need to implement some methods. If so, it's better that this can be changed by flag also.

Reply:
You mean that these new parameters should be in `tfnotify.yaml`, right?

Currently, plan and apply's `title` and `message` cannot be configured in this configuration. If we add these parameters only for destroy warning, it will be inconsistent.

Reply:
>You mean that these new parameters should be in tfnotify.yaml, right?

Yeah,  I meant this. But ...

Hmm, agreed.